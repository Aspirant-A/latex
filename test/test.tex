\documentclass{ctexart}
\usepackage{amsfonts}
% \renewcommand{\baselinestretch}{1.25}
% \ctexset{
%     section={format={\heiti \zihao{4}}},
%     subsection={format={\heiti \zihao{6} \bfseries}, beforeskip=0pt, afterskip=0pt },
%     subsubsection={format={\kaishu \zihao{5}}, beforeskip=0pt, afterskip=0pt},
% }

% \usepackage[colorlinks, urlcolor=black, linkcolor=black]{hyperref}

% \usepackage[left=2.5cm,right=1.97cm,top=2.5cm,bottom=2.5cm]{geometry}

% \usepackage{fontspec}
% \setmainfont{Times New Roman}
% \usepackage{ctex}

% \documentclass{book}
% \usepackage{amsmath}


% \title{\heiti \zihao{2}用 \LaTeX 排版一篇中文期刊论文}
% \author{\kaishu \zihao{-4}赵竞昂}
% \date{\today}


\begin{document}
ojaidsfhoa
% \maketitle
% \tableofcontents
% \input{ab.tex}
% \input{first.tex}
% \input{two.tex}
% \input{three.tex}

% 附录：
% 	\begin{enumerate}
% 	\item 导入章节input 以及 include 的用法及区别：\\		
% 		\url{https://blog.csdn.net/weixin_42919606/article/details/82939495}	
% 	\item  标题格式更改\\		
% 		\url{https://www.jianshu.com/p/d7848f815e5f}		
% 	\item 编号方法itemize 和 enumerate ：\\		
% 		\url{http://blog.sina.com.cn/s/blog_77f5a65c0101fmjl.html}\\		
% 		\url{https://blog.csdn.net/qq_18055167/article/details/83714725}		
% 	\item \label{字体字号} 字体字号及相关设置：\\		
% 		\url{https://blog.csdn.net/weixin_44537194/article/details/87720878}\\	
% 		\url{https://blog.csdn.net/ujsdui/article/details/79075327}
% 	\item 常用命令：\\		
% 		\url{https://blog.csdn.net/garfielder007/article/details/51646802}
% 	\item 文档类型以及页面设置的基本介绍：\\
% 		\url{https://blog.csdn.net/wei_love_2017/article/details/86617235}
% 	\end{enumerate}	
	
	



\end{document}

% \title{\LaTeX{} 学习手册}
% \author{赵竞昂}
% \maketitle

% \tableofcontents

% \section{第一节}
% \subsection{小结}
% \subsection{小结}
% \subsection{小结}
% \section{第二节}
% \subsection{小结}
% \subsection{小结}
% \subsection{小结}
% \section{第三节}
% \subsection{小结}
% \subsection{小结}
% \subsection{小结}
% \section{第四节}
% \subsection{小结}
% \subsection{小结}
% \subsection{小结}
% \section{第五节}
% \subsection{小结}
% \subsection{小结}
% \subsection{小结}
% \subsection{小结}
%段落缩进
% \parindent=0pt

%下划线
% \usepackage[normalem]{ulem} 
% \usepackage{xeCJKfntef}

%颜色
% \usepackage{xcolor}

% \begin{document}
% \begin{flushleft}
%     \hspace{1em} 在$\bigtriangleup OFP$，由三边关系，得$OP+OF>FP$,\\
%     由于$OP$和$OF$为定值，所以若$O,F,P$构成三角形，\\
%     那么一定会有$OP+OF>FP$，但是在旋转的过程\\
%     中会出现$O,F,P$共线,此时会有$OP+OF=FP$，所\\
%     以，此时$FP$会有最大值。
% \end{flushleft}
% \begin{flushleft}
%     \hspace{1em} 此时，大家一定能注意到，若$P$继续运动\\
%     一定会出现$O,F,P$再次共线,在这段运动过程中，\\
%     我能由三边关系，得$OF-OP<FP$，那我\\
%     想问什么时候$FP$会有最小值呢，此时大家肯定\\
%     会告诉我在下一次$O,F,P$共线时，因为此时会\\
%     有$OF-OP=FP$，所以，此时$FP$会有最小值。
% \end{flushleft}
% \begin{center}
%     \uline{圆外一点到圆周上一动点的距离：\\
%     {\color{blue}最大值为定点到圆心的距离加半径，}\\
%     最小值为定点到圆心的距离减半径。}
% \end{center}

% \begin{center}
%     若大家没看懂，那就让你们常老师教你们哟！！！
% \end{center}

% 在\LaTeX{}中排版中文。
% 汉字和English单词混排，通常不需要在中英文之间添加额外的空格。
% 当然，为了代码的可读性，加上汉字和 English 之间的空格也无妨。
% 汉字换行时不会引入多余的空格。

%换行命令\par 也可以用空行来代替
% Several spaces equal one.

% Front spaces are ignored.

% An empty line starts a new
% paragraph.\par
% A \verb|\par| command does
% the same.

% \end{document} 