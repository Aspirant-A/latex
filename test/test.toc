\contentsline {section}{\numberline {1}页面格式设置}{1}{section.1}%
\contentsline {section}{\numberline {2}基本文本格式以及段落调整}{1}{section.2}%
\contentsline {subsection}{\numberline {2.1}字体字号}{2}{subsection.2.1}%
\contentsline {subsubsection}{\numberline {2.1.1}字体}{2}{subsubsection.2.1.1}%
\contentsline {subsubsection}{\numberline {2.1.2}字号}{2}{subsubsection.2.1.2}%
\contentsline {section}{\numberline {3}段落调整}{2}{section.3}%
\contentsline {subsection}{\numberline {3.1}基本段落格式}{2}{subsection.3.1}%
\contentsline {subsection}{\numberline {3.2}行间距调整}{3}{subsection.3.2}%
\contentsline {subsection}{\numberline {3.3}自动编号}{3}{subsection.3.3}%
\contentsline {section}{\numberline {4}多级标题以及文件导入}{3}{section.4}%
\contentsline {subsection}{\numberline {4.1}多级标题格式设置}{3}{subsection.4.1}%
\contentsline {subsection}{\numberline {4.2}文件导入}{3}{subsection.4.2}%
