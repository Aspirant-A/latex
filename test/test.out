\BOOKMARK [1][-]{section.1}{\376\377\230\165\227\142\150\074\137\017\213\276\177\156}{}% 1
\BOOKMARK [1][-]{section.2}{\376\377\127\372\147\054\145\207\147\054\150\074\137\017\116\345\123\312\153\265\204\075\214\003\145\164}{}% 2
\BOOKMARK [2][-]{subsection.2.1}{\376\377\133\127\117\123\133\127\123\367}{section.2}% 3
\BOOKMARK [3][-]{subsubsection.2.1.1}{\376\377\133\127\117\123}{subsection.2.1}% 4
\BOOKMARK [3][-]{subsubsection.2.1.2}{\376\377\133\127\123\367}{subsection.2.1}% 5
\BOOKMARK [1][-]{section.3}{\376\377\153\265\204\075\214\003\145\164}{}% 6
\BOOKMARK [2][-]{subsection.3.1}{\376\377\127\372\147\054\153\265\204\075\150\074\137\017}{section.3}% 7
\BOOKMARK [2][-]{subsection.3.2}{\376\377\210\114\225\364\215\335\214\003\145\164}{section.3}% 8
\BOOKMARK [2][-]{subsection.3.3}{\376\377\201\352\122\250\177\026\123\367}{section.3}% 9
\BOOKMARK [1][-]{section.4}{\376\377\131\032\176\247\150\007\230\230\116\345\123\312\145\207\116\366\133\374\121\145}{}% 10
\BOOKMARK [2][-]{subsection.4.1}{\376\377\131\032\176\247\150\007\230\230\150\074\137\017\213\276\177\156}{section.4}% 11
\BOOKMARK [2][-]{subsection.4.2}{\376\377\145\207\116\366\133\374\121\145}{section.4}% 12
